build/avalanchego-apis/index-api.md

```diff
@@ -0,0 +1,313 @@
+# Index API
```

Comment:
This file has been moved to a new location on /apis/avalanchego/apis/index-api. Please update that doc accordingly.